.circleci/config.yml

```diff
@@ -516,20 +722,13 @@ jobs:
           fingerprints:
             - 6f:4b:55:76:0e:cd:27:7d:ad:c3:28:38:53:69:5c:32
       - checkout
-      - run:
-          name: Cancel running builds
-          command: |
-            python cancel_redundant_builds.py \
-            --circle_project_username="${CIRCLE_PROJECT_USERNAME}" \
-            --circle_project_reponame="${CIRCLE_PROJECT_REPONAME}" \
-            --circle_branch="${CIRCLE_BRANCH}" \
-            --circle_builds_token="${CIRCLE_BUILDS_TOKEN}" \
-            --terms_to_waitfor="apply,plan" \
-            --prod_job_terms=" production,shared-production"
-          working_directory: ~/project/ecs_helper/cmd/circle_builds
       - run:
           name: Set environment
           command: ~/project/.circleci/set_env.sh >> $BASH_ENV
+      - run:
+          name: Download common passwords file
+          command: wget -q -O commonpasswords.txt "https://www.ncsc.gov.uk/static-assets/documents/PwnedPasswordsTop100k.txt"
+          working_directory: ~/project/client/extra
```

Comment:
Cant tell from the wget flags but is this now not failing when we get a 500 and continuing with the build?

Reply:
It's quiet but still does a non zero error code. I will make it non quiet anyway as you're right, there's nothing insecure in there and it would be good to see the failure!